### ISSUE TITLE
Several Issues

### ISSUE THREAD

**Issue by cutesniper22** [piyushgarg/rclone-mount#5]
Hi, I'm quite interested with your hard work.
However, I'm facing some problem with your rclone-mount magisk module.
1. My device will begin to freeze after successfully boot to lockscreen. I realized that my rclone.conf was encrypted. This issue no longer happened once i replace with decrypted conf file. (Encrypted conf file cause freeze?)

2.I try to key in this "rclone-mount" command, it showed cmd not found in terminal. I found out it did not exist in physical system path "/system/xbin".  (Attached with screenshot)

3. Can further explain on installation steps especially on custom param. Can you come out some examples on this configuration... 

Appreciate your help. thank you.

Device info
Xiaomi mi 9 
MIUI 10.2.9.0 (Android Pie)
![Screenshot_2019-07-06-00-19-53-541_com Z mq8Zfdr H wz8a](https://user-images.githubusercontent.com/26847827/60734906-18acfb80-9f84-11e9-8bce-bc72a737eb8b.png)
![Screenshot_2019-07-06-00-21-07-679_com jrummy root browserfree](https://user-images.githubusercontent.com/26847827/60734912-1c408280-9f84-11e9-81e9-46298c44047d.png)

**Comment by CodeNameT1M** [comment]
Have you tried navigating to /data/adb/modules_update/com.piyushgarg.rclone/rclone? Check if the Binaries and the rclone-mount script are in that directory.

![image](https://user-images.githubusercontent.com/28508724/60739783-7ed55680-9f63-11e9-8b15-601a82f1caec.png)

**Comment by Geofferey** [comment]
The  `rclone-mount` command will be replaced with `rclone remount` in v1.4. I will also include some examples or links to issues that are examples in README.md in regards to custom params. As for the encrypted file I'm guessing support for those are a no go for now. Was just the password encrypted or whole file?

**Comment by cutesniper22** [comment]
> Have you tried navigating to /data/adb/modules_update/com.piyushgarg.rclone/rclone? Check if the Binaries and the rclone-mount script are in that directory.
> 
> ![image](https://user-images.githubusercontent.com/28508724/60739783-7ed55680-9f63-11e9-8b15-601a82f1caec.png)

Yup. just checked. As you said, the rclone directory already there, but still typed "rclone-mount" command was  not working in terminal.
![Screenshot_2019-07-06-11-06-29-501_com jrummy root browserfree](https://user-images.githubusercontent.com/26847827/60750662-86861100-9fde-11e9-9698-5425b5cd0e5f.png)

**Comment by cutesniper22** [comment]
> The `rclone-mount` command will be replaced with `rclone remount` in v1.4. I will also include some examples or links to issues that are examples in README.md in regards to custom params. As for the encrypted file I'm guessing support for those are a no go for now. Was just the password encrypted or whole file?

Thanks for reply. The whole file is encrypted. I'm not urgent with encryption support. As long as it is runnable then should be fine.
e.g: 

![image](https://user-images.githubusercontent.com/26847827/60750683-ea103e80-9fde-11e9-85bc-3627d084e70b.png)

**Comment by Geofferey** [comment]
@cutesniper22 No problem. If you use file based encryption your .conf files will be safe on SD 'til unlock. I might look into it a lil later but it's not urgent for me either ATM. I am working on some doc. You can check it out but it's nowhere near ready. https://github.com/Magisk-Modules-Repo/com.piyushgarg.rclone/issues/2 Enjoy the mod and keep your eyes peeled for 1.4 release. 😁

**Comment by moeacgx** [comment]
请问为什么安装很久都没安装好?
